tests/test_utils.py

```diff
@@ -0,0 +1,86 @@
+"""Test utils"""
+# pylint: disable=invalid-name
```

Comment:
not necessary. remove

Reply:
Unfortunately, pylint is really picky and don't understand that this is a script. It complains for instance about variables not being purely upper case. Some of the tools have their own non-standard, non-PEP8 notion of best practices that I do not always agree with. The non-80 character line length (that is used by all large python projects and is described in PEP8 and the google style guidelines) is one example. Another is this requirement that all variable names in a test script must be purely upper case.